myapp/spec/system/task_sorting_func_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Task Sorting', type: :system do
+  before do
+    # Create a user because we have foreign key
+    create(:user)
+  end
+
+  it 'check sort functionality by items as latest first & oldest first' do
```

Comment:
I prefer these test cases are written separately
```ruby
it 'check sort functionality by items as latest first' do
  ...
  expect(task_titles_only).to eq(['Test Task latest one', 'Test Task created on second', 'Test Task first one'])
end

it 'check sort functionality by items as oldest first' do
  ...
  expect(task_titles_only).to eq(['Test Task first one', 'Test Task created on second', 'Test Task latest one'])
end
```

Reply:
Basically we should avoid writing multiple `expect` in one `it` block. Sometimes we use multiple `expect` but test code look like this in that case:
```ruby
it 'expects multiple things' do
 do_something_like_clicking_link
 expect(foo).to eq 'bar'
 expect(bar).to eq 'baz'
end
# write multiple expectation at the end of the block
```

Reply:
separated

Reply:
> Basically we should avoid writing multiple `expect` in one `it` block. Sometimes we use multiple `expect` but test code look like this in that case:
> 
> ```ruby
> it 'expects multiple things' do
>  do_something_like_clicking_link
>  expect(foo).to eq 'bar'
>  expect(bar).to eq 'baz'
> end
> # write multiple expectation at the end of the block
> ```

For the `task_spec.rb` i left as it was but for the `task_sorting_func_spec.rb` I made the separation. 
so please check once again

Thanks